shared/chat/conversation/messages/wrapper/chat-send.js

```diff
@@ -75,12 +75,23 @@ class SendIndicator extends React.Component<Props, State> {
     }
   }
 
+  _restart() {
+    if (shownEncryptingSet.has(this.props.id)) {
```

Comment:
No need to check if it's already in -- simpler to remove the `if` and do:
```
shownEncryptingSet.delete(this.props.id)
```

Reply:
actually, hmm, why remove it from `shownEncryptingSet`? You're starting the animation from `sending` anyway, right?

Reply:
Yep you're right, that should be removed